### ISSUE TITLE
Magento 2.3.3 Role Resource Tree Not Showing

### ISSUE THREAD

**Issue by ghost** [magento/magento2#25610]
### Preconditions
1. Magento 2.3.3
2. no third-party module

### Steps to reproduce
1. In the backend, going to system->roles, adding a new role and setting it to custom permissions, the resource tree is empty

### Expected result
1. resource tree not empty

### Actual result
1. 
![Schermata 2019-11-14 alle 16 46 35](https://user-images.githubusercontent.com/48567585/68885205-005ca280-0715-11ea-8dde-90c51600bf08.png)

2. 
![Schermata 2019-11-14 alle 16 52 01](https://user-images.githubusercontent.com/48567585/68885218-06528380-0715-11ea-805b-16d310e3e0f6.png)

**Comment by ghost** [comment]
@magento give me 2.3.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @gfigliolia. Thank you for your request. I'm working on Magento 2.3.3-develop instance for you

**Comment by drpayyne** [comment]
@magento give me 2.3.3 instance

**Comment by magento-engcom-team** [comment]
Hi @drpayyne. Thank you for your request. I'm working on Magento 2.3.3 instance for you

**Comment by magento-engcom-team** [comment]
Hi @drpayyne, here is your [Magento instance](https://i-25610-2-3-3.instances.magento-community.engineering).
Admin access: [https://i-25610-2-3-3.instances.magento-community.engineering/admin](https://i-25610-2-3-3.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by drpayyne** [comment]
Hi @gfigliolia, I'm not able to reproduce this issue on a `2.3.3` vanilla instance. I can view the role tree structure while creating a new role. 

![Screenshot from 2019-11-15 01-29-10](https://user-images.githubusercontent.com/22623301/68891685-5e09e280-0747-11ea-84c0-bb7ea825f64d.png)

**Comment by drpayyne** [comment]
Closing this as I'm unable to reproduce it. Please reopen in case of queries. Thanks!

**Comment by joris-ati4** [comment]
@gfigliolia , I stumbled on the same problem as you. The problem I found is that my admin was in french and a single quote in the translation "Registre d'actions" was transformed to a double quote. 

I see your back-end is in spanish so maybe you have the same problem. To debug, it's quite easy: 

1. Open your Dev Tools
2. Put a JS debugger in the Sources panel on Line 52 of file `adminhtml/Magento/backend/fr_FR/mage/apply/main.js`
3. Skip through the steps until you see the error in the console. 
4. Repeat your step and verify which element causes the error. Mine one the mage-init of `<div class="tree x-tree jstree jstree-0 jstree-focused jstree-default" data-role="resource-tree">`
5. The problem was the double quote that broke the JSON string...

To fix the bug I removed the single quote from the translation file. 

@drpayyne : I didn't had the time to check but could it be possible that Magento converts the single quote into a double quote ? 

Have a nice evening !
